core/src/main/java/de/danoeh/antennapod/core/service/playback/PlaybackServiceNotificationBuilder.java

```diff
@@ -116,7 +117,7 @@ public Notification build() {
                 NotificationUtils.CHANNEL_ID_PLAYING);
 
         if (playable != null) {
-            notification.setContentTitle(playable.getFeedTitle());
+            notification.setContentTitle(Html.fromHtml("<b>" + playable.getFeedTitle() + "</b>"));
```

Comment:
To be honest, I would keep the default here. Makes it more consistent with the rest of the system.

Reply:
Well, I do not mind. Then this pull request would only change some strings :wink:

Reply:
Making strings more clear is a good thing, too :)